package.json

```diff
@@ -1,17 +1,33 @@
 {
-  "name": "",
+  "name": "reviews_module",
   "version": "",
   "description": "",
+  "main": "index.js",
   "author": "",
   "license": "",
   "engines": {
     "node": ">=6.13.0"
   },
   "scripts": {
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/index.js",
+    "start": "nodemon server/index.js"
```

Comment:
I think this is the script the deployed version will use, so start may want to use just the `node` command and not `nodemon`